access/grpc/client.go

```diff
@@ -42,20 +43,50 @@ const CanarynetHost = "access.canary.nodes.onflow.org:9000"
 const MainnetHost = "access.mainnet.nodes.onflow.org:9000"
 const PreviewnetHost = "access.previewnet.nodes.onflow.org:9000"
 
+// ClientOption is a configuration option for the client.
+type ClientOption func (*options) 
+
+type options struct {
+	dialOptions []grpc.DialOption
+	jsonOptions []cdcjson.Option
+}
+
+func DefaultClientOptions() *options {
+	return &options{
+		dialOptions: []grpc.DialOption{grpc.WithTransportCredentials(insecure.NewCredentials())},
+		jsonOptions: []cdcjson.Option{cdcjson.WithAllowUnstructuredStaticTypes(true)},
+	}
+}
+
+// WithGRPCDialOptions wraps a grpc.DialOption into a ClientOption.
+func WithGRPCDialOptions(dialOpts ...grpc.DialOption) ClientOption {
```

Comment:
these are modelled after SubscribeOptions, seen here
https://github.com/onflow/flow-go-sdk/blob/e56a70a692d1a6507ccc023032f67159ee923abc/access/grpc/grpc.go#L50-L73

Reply:
This is great!